ethers-providers/src/lib.rs

```diff
@@ -756,16 +758,14 @@ pub mod test_provider {
     }
 
     impl TestProvider {
-        pub fn new(keys: &'static [&'static str], network: &str) -> Self {
-            Self { keys: Mutex::new(keys.iter().cycle()), network: network.to_owned() }
+        pub fn new(network: impl Into<String>) -> Self {
```

Comment:
i'd like to keep this around so i can specify providers w/ non infura keys in the future